### PR TITLE
Add reports landing page

### PR DESCRIPTION
**Pull request by PaulMorris**
Add a 'Reports' landing page and a page for 'Applications ~~Approved and Denied~~ by Reviewer' report.  

This is WIP with changes made on top of those in PR #800.  Review can happen after #800 lands.

There is more to do on the CSS side to fix the date picker icon placement, and to get the date inputs to look consistent with the other date inputs.  

I've set a `border-radius` of 6px on the div with the rounded corners.  Elsewhere in the site these rounded corners are drawn via tiny images in the HTML (a legacy practice, eventually we should remove these in favor of current CSS approach).  I checked that adding the border-radius wasn't visible in existing places where these tiny images are used. (It merges with the tiny images.)  However, some of the existing places only have the rounded corners on say the top corners.  To my eye it seems preferable and simpler to have the rounded corners on all these corners everywhere.  So while this changes the appearance slightly, I'd call it an improvement.  But I wanted to mention it.

Screenshots:

screenshot-2018-5-3 reports 2

screenshot-2018-5-3 reports 3

Issue #781: Create Report for users approving... applications

### CODE REVIEW

**Comment by PaulMorris** [comment]
I improved the styling/CSS to put the date picker icon in the right place and make the inputs look like others in the PSM.  Also renamed the report to "Applications Approved and Denied" (which seems clearer than my previous version with "Approvals and Denials").  I've also changed "report" in the URL to "reports" to match the UI, and put the reports JSP files in a "reports" directory.

screenshot-2018-5-4 applications approved and denied

File: psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_approved_denied.jsp [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<%@ include file="/WEB-INF/pages/admin/includes/taglibs.jsp" %>
+<!DOCTYPE html>
+<html lang="en-US">
+  <c:set var="title" value="Applications Approved and Denied"/>
```
Comment (cecilia-donnelly) [inline psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_approved_denied.jsp]: You asked about this in Zulip, @PaulMorris -- I think the purpose of this report is to see the *reviewer*, so maybe "Applications by Reviewer."

File: psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_approved_denied.jsp [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                      id="endDate"
+                      class="date hasDatePicker inputBox"
+                      title="Date End"
+                      placeholder="End Date"
+                      class="date"
+                      type="text"
+                      readonly="readonly"
+                    />
+                  </span>
+                </div>
```
Comment (cecilia-donnelly) [inline psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_approved_denied.jsp]: The datepickers still don't show up inside the input elements for me, in Firefox: 
screenshot-2018-5-4 applications approved and denied

File: psm-app/cms-web/src/main/java/gov/medicaid/controllers/admin/ReportController.java [hunk 3]
```diff
@@ -7,8 +7,16 @@
...
+    public ModelAndView viewReports() {
+        return new ModelAndView("admin/reports/reports");
+    }
+
+    @RequestMapping(
+            value = "/admin/reports/applications-approved-denied",
+            method = RequestMethod.GET
+    )
+    public ModelAndView viewApprovedDenied() {
+        return new ModelAndView("admin/reports/applications_approved_denied");
```
Comment (cecilia-donnelly) [inline psm-app/cms-web/src/main/java/gov/medicaid/controllers/admin/ReportController.java]: For consistency we should also change this to "applications by reviewer" or similar.

File: psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_approved_denied.jsp (hunk 2 above)
Comment (cecilia-donnelly) [reply psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_approved_denied.jsp]: Ah, sorry, you mentioned in the PR message that there's more to do here.

**Comment by PaulMorris** [comment]
Now rebased on current master and ready to review.  The report is now called "Applications by Reviewer" as requested by @cecilia-donnelly .  Includes a little refactoring of the integration tests (if you want to take a look at that @frankduncan ).

File: psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_approved_denied.jsp (hunk 2 above)
Comment (PaulMorris) [reply psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_approved_denied.jsp]: Hmmm, I wonder if this is a CSS caching issue?  See if refreshing the page helps if it shows up again in the latest version.

File: psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_by_reviewer.jsp [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+      <div id="mainContent">
+        <div class="contentWidth">
+          <div class="mainNav">
+            <h:handlebars template="includes/logo" context="${pageContext}"/>
+            <h:handlebars template="includes/nav" context="${pageContext}"/>
+          </div>
+          <div class="breadCrumb">
+            <a href="<c:url value='/admin/reports' />">Reports</a>
+            <span>Applications by Reviewer</span>
+          </div>
```
Comment (cecilia-donnelly) [inline psm-app/cms-web/WebContent/WEB-INF/pages/admin/reports/applications_by_reviewer.jsp]: Nice real breadcrumb!

**Pull request merged**